src/Transports.AspNetCore.NewtonsoftJson/GraphQLRequestDeserializer.cs

```diff
@@ -31,7 +31,7 @@ public Task<GraphQLRequestDeserializationResult> DeserializeFromJsonBodyAsync(Ht
             // work except for the disposing inner stream.
             var reader = new StreamReader(httpRequest.Body);
 
-            var result = new GraphQLRequestDeserializationResult() { WasSuccessful = true };
+            var result = new GraphQLRequestDeserializationResult() { IsSuccessful = true };
```

Comment:
```suggestion
            var result = new GraphQLRequestDeserializationResult { IsSuccessful = true };
```

One more my traditional suggestion.

Reply:
:P